compiler/src/org.graalvm.compiler.lir/src/org/graalvm/compiler/lir/gen/LIRGenerator.java

```diff
@@ -88,6 +90,8 @@
 
     public static class Options {
         // @formatter:off
+        @Option(help = "", type = OptionType.Expert)
```

Comment:
New options must not have an empty `help` value. That said, it would be best to avoid adding this option altogether. Can you not tune the heuristic to choose the hashing strategy?